### DISCUSSION TITLE
Homer API comprehensive doc

### DISCUSSION THREAD

**Discussion by drdoom1976** [sipcapture/homer-app#543] [Q&A] [answered]
Hi.

I'm looking for some in depth documentation for the Homer-App API.  I used the doc browser in the GUI, but there is still some questions being unanswered.

As an example, if I would like to retrieve the number of calls terminated by a CANCEL SIP method for a given time frame, my first guess will be to use the /search/call/data request, ins't it ?

1) Can the API returns a list of calls filtered by the SIP Method ?
2) into the search API call, there is a search field: "search": "`{\"1_call\":[{\"name\":\"limit\",\"value\":\"10\",\"type\":\"string\",\"hepid\":1}]}
is that field is mandantory, and if so, how should I use it ?

Thanks.

**Comment by lmangani** [comment, marked as answer]
The Swagger API is documented in homer-app via the admin menu. The best way to get started is looking at the UI API interactions.
The type (call registration, other) is mandatory.
